grammar.js

```diff
@@ -723,6 +724,59 @@ module.exports = grammar({
       ),
     ),
 
+    comment_text: $ => $._literal_string,
+
+    comment_statement: $ => seq(
+      $.keyword_comment,
+      $.keyword_on,
+      $._comment_target,
+      $.keyword_is,
+      choice(
+        $.keyword_null,
+        $.comment_text,
+      ),
+    ),
+
+    _comment_target: $ => choice(
+      // TODO: access method
+      // TODO: aggregate
+      $.cast,
+      // TODO: collation
+      seq($.keyword_column, $.object_reference),
```

Comment:
```suggestion
      seq($.keyword_column, alias($._qualified_field, $.object_reference)),
```
`comment on column schema.table.column is 'asdf';` wouldn't parse since `object_reference` only covers up to two parts. `_qualified_field` technically allows a parenthesized schema+table but I don't think we need to add a whole new node type for tripartite names over this